tests/firedrake/adjoint/test_burgers_newton.py

```diff
@@ -123,17 +129,23 @@ def J(ic, solve_type, timestep, steps, V):
 
 @pytest.mark.skipcomplex
 @pytest.mark.parametrize("solve_type, checkpointing",
-                         [("solve", "Revolve"), ("NLVS", "Revolve"),
-                          ("solve", "SingleMemory"), ("NLVS", "SingleMemory"),
-                          ("solve", "NoneAdjoint"), ("NLVS", "NoneAdjoint"),
-                          ("solve", "Mixed"), ("NLVS", "Mixed"),
-                          ("solve", None), ("NLVS", None)])
-def test_burgers_newton(solve_type, checkpointing):
+                         [("solve", "Revolve"),
```

Comment:
I think you want to have two `parametrize` things here:
```py
@pytest.mark.parametrize("solve_type", ["solve", "NLVS"])
@pytest.mark.parametrize("checkpointing", ["Revolve", "SingleMemory", "NoneAdjoint", "Mixed", None])
```

Reply:
done

Reply:
I think you did it below but not here?

Reply:
Oh yes. I just made it.